src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/EditingTargetCodeExecution.java

```diff
@@ -163,18 +163,7 @@ public void executeSelection(boolean consoleExecuteWhenNotFocused,
       // advance if there is no current selection
       if (noSelection && moveCursorAfter)
       {
-         if (DocumentMode.isCursorInPythonMode(docDisplay_))
```

Comment:
This extra logic is no longer necessary as the REPL provided by `reticulate` is smart enough to handle this correctly now.